dashboard/src/components/JobQueueItem.tsx

```
@@ -0,0 +1,74 @@
+  const handleClick = () => onSelect(workflow.id);
+
+  const handleKeyDown = (e: React.KeyboardEvent) => {
+    if (e.key === 'Enter' || e.key === ' ') {
+      e.preventDefault();
+      onSelect(workflow.id);
+    }
+  };
+
+  return (
+    <div
+      role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)